### PR TITLE
Migrate to Swift 5

### PR DESCRIPTION
**Pull request by carol-mohemian**
Needed to resolve problems while migrating MPC.

### CODE REVIEW

File: Source/SideMenuController.swift [hunk 1]
```diff
@@ -132,7 +132,7 @@ public extension SideMenuController {
     // MARK: Public
     
     open weak var delegate: SideMenuControllerDelegate?
-    open static var preferences: Preferences = Preferences()
+    public static var preferences: Preferences = Preferences()
```
Comment (ghost) [inline Source/SideMenuController.swift:135]: No more `open` in Swift 5?

Comment (carol-mohemian) [reply Source/SideMenuController.swift:135]: Apparently, I think it has to do with declaring it static. So `open` isn't gone but it crashes with static I think.

**Review by markus-mohemian — APPROVED** [review]
✅ LGTM

**Comment by ghost** [comment]
👍

**Pull request merged**